testsuite/tests/runtime-C-exceptions/test.ml

```diff
@@ -0,0 +1,11 @@
+external failwith_from_ocaml : string -> 'a = "caml_failwith_value"
```

Comment:
This is maybe a bit naughty, because I don't think that function is supposed to be called from OCaml.  However I imagine it's probably ok in a test case.